giskard/llm/talk/tools/predict.py

```diff
@@ -0,0 +1,153 @@
+import pandas as pd
+from thefuzz import fuzz
+
+from giskard.datasets.base import Dataset
+from giskard.llm.talk.config import ToolDescription
+from giskard.llm.talk.tools.base import BaseTool, get_feature_json_type
+
+
+class PredictTool(BaseTool):
+    """Prediction calculation Tool.
+
+    Attributes
+    ----------
+    default_name : str
+        The default name of the Tool. Can be re-defined with constructor.
+    default_description: str
+        The default description of the Tool's functioning. Can be re-defined with constructor.
+    """
+
+    default_name: str = "predict"
+    default_description: str = ToolDescription.PREDICT.value
+
+    @property
+    def specification(self) -> str:
+        """Return the Tool's specification in a JSON Schema format.
+
+        Returns
+        -------
+        str
+            The Tool's specification.
+        """
+        feature_json_type = get_feature_json_type(self._dataset)
+
+        return {
+            "type": "function",
+            "function": {
+                "name": self.name,
+                "description": self.description,
+                "parameters": {
+                    "type": "object",
+                    "properties": {
+                        "features_dict": {
+                            "type": "object",
+                            "properties": {
+                                feature: {"type": dtype} for feature, dtype in list(feature_json_type.items())
+                            },
+                        }
+                    },
+                    "required": ["features_dict"],
+                },
+            },
+        }
+
+    def _get_input_from_dataset(self, row_filter: dict) -> pd.DataFrame:
+        """Get input from dataset.
+
+        Filter rows from the dataset, using the `row_filter`.
+
+        Parameters
+        ----------
+        row_filter : dict
+            The dictionary with features and related values to filter the dataset.
+
+        Returns
+        -------
+        pd.DataFrame
+            The DataFrame with filtered rows.
+        """
+        threshold = 85
```

Comment:
Should be defined as a constant at list, in the beginning of the file

Reply:
I did not get, how you see that. I decided to import this constant from the `llm.talk.config`